src/Tar/project.json

```diff
@@ -1,16 +1,21 @@
 ﻿{
   "version": "0.1.0",
   "frameworks": {
+    "netstandard1.6": {
+      "dependencies": {
+        "System.IO": "4.1.0",
+        "System.IO.FileSystem": "4.0.1",
+        "System.Linq": "4.1.0",
+        "System.Runtime": "4.1.0",
+        "System.Runtime.Extensions": "4.1.0",
+        "System.Text.Encoding": "4.0.11",
+        "System.Text.Encoding.Extensions": "4.0.11",
+        "System.Threading.Tasks": "4.0.11"
+      }
```

Comment:
Not sure what you mean? If you just want to get the standard set you can just say `"NETStandard.Library": "1.6.0"`. We know thats what PS does in the SDK spec so that would be ok here.